android/app/src/main/java/com/mattermost/rnbeta/CustomPushNotification.java

```diff
@@ -262,23 +304,39 @@ private static void saveNotificationsMap(Context context, Map<String, List<Integ
         }
     }
 
-    private static Map<String, List<Integer>> loadNotificationsMap(Context context) {
-        Map<String, List<Integer>> outputMap = new HashMap<>();
+    /**
+     * Map Structure
+     * 
+     * {
+     *     channel_id1: {
```

Comment:
Why did you changed the structure we discussed?

Reply:
I still haven't made the changes after the discussion. I was thinking about the use cases with/without CRT (if someone toggles). It's about adding more conditions and I want to see which structure makes it more readable.